ext/crypto/encrypt.rs

```diff
@@ -136,3 +161,54 @@ fn encrypt_aes_cbc(
   };
   Ok(ciphertext)
 }
+
+fn encrypt_aes_ctr_gen<B, F>(key: &[u8], counter: &[u8], data: &[u8]) -> Vec<u8>
+where
+  B: BlockEncrypt + NewBlockCipher,
+  F: CtrFlavor<B::BlockSize>,
+{
+  let mut cipher = Ctr::<B, F>::new(key.into(), counter.into());
+
+  let mut ciphertext = data.to_vec();
+  cipher.apply_keystream(&mut ciphertext);
```

Comment:
Just checking to see if my understanding is correct: it's okay to use `apply_keystream()` instead of `try_apply_keystream()` because the ciphertext is always a multiple of the block size?

Reply:
As a stream cipher, any length of text is allowed, not being restricted to multiples of the block-size. There is a case where `apply_keystream` will throw, however, which is when the user tries to encrypt more than `2 ^ length x 16` bytes of data, which in our current worse case, is 128GB in a single call.

Altered to use `try_apply_keystream` as excess of caution.